docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 x-postgres-common:
   &postgres-common
-  image: postgres:13.8
+  image: ${POSTGRES_IMAGE:-postgres:13.8}
```

Comment:
Seems like it's going twice?
`${POSTGRES_IMAGE:-postgres:13.8}`
`${{ secrets.DOCKER_REGISTRY }}/doma/utils/postgres:13.2:-postgres:13.8`

Reply:
Why versions mismatched?

Reply:
And can I run it locally without envs?

Reply:
No it's not like that.
According to official docker docs `:-` is special marker for "default value" if no custom env variable was provided by user

Reply:
So, if you don't set any env's, you have `postgres:13.8` as your image. Otherwise image value would be same as `POSTGRES_IMAGE`